resources/views/community/contact.blade.php

```diff
@@ -46,9 +46,37 @@
                     <li>Getting involved in a QA sub-team.</li>
                 </ul>
             </x-section>
+
+            <x-section>
+                <h2>DevQuest</h2>
+
+                <p class="mb-3">
+                    <a href='/createmessage.php?t=DevQuest'>Send a message to DevQuest</a> for
+                    submissions, questions, ideas, or reporting issues related to
+                    <a href='/game/5686'>DevQuest</a>.
+                </p>
+            </x-section>
         </div>
 
         <div>
+            <x-section>
+                <h2>Cheating Reports</h2>
+
+                <p class="mb-3">
+                    <a href="/createmessage.php?t=RACheats">Send a message to RACheats</a>
+                    if you believe someone is in violation of our
+                    <a href="https://docs.retroachievements.org/Global-Leaderboard-and-Achievement-Hunting-Rules/#not-allowed">Global Leaderboard and Achievement Hunting Rules</a>.
```

Comment:
Did you mean to link this to the `#not-allowed` subsection instead of just linking to the top of the page?

Reply:
Yep, that was intentional as I figured it's the precise criteria users should be DMing the RACheats account for.